packages/common/src/content/_internal/computeProps.ts

```diff
@@ -59,6 +66,12 @@ export function computeProps(
   // when we receive a schedule from the enrichments, we want to use it, otherwise default to automatic
   content.schedule = enrichments.schedule;
 
+  // calculate extendedProps
+  content.extendedProps = isService(content.url)
+    ? getServiceExtendedProps(model.item, enrichments, requestOptions)
+    : getContentExtendedProps(model.item, enrichments, requestOptions);
+
+  // TODO: Remove once .serverQueryCapability, .serverExtractCapability, and .serverExtractFormats are removed
```

Comment:
highlighting for visibility

Reply:
Here's the follow-on tech ticket that encapsulates this: https://devtopia.esri.com/dc/hub/issues/10901